rebar.config

```diff
@@ -20,17 +20,17 @@
 
 {ex_doc, [
     {extras, [
-          {'CHANGELOG.md', #{title => <<"Changelog">>}},
-          {'README.md', #{title => <<"Overview">>}},
-          {'LICENSE.md', #{title => <<"License">>}}
+          {"CHANGELOG.md", [{title, ~B"Changelog"}]},
+          {~B"README.md", [{title, ~B"Overview"}]},
+          {~B"LICENSE.md", [{title, ~B"License"}]}
     ]},
-    {main, <<"readme">>},
-    {homepage_url, <<"https://github.com/starbelly/rebar3_ex_doc">>},
-    {source_url, <<"https://github.com/starbelly/rebar3_ex_doc">>},
-    {assets, <<"assets">>},
+    {main, ~B"readme"},
```

Comment:
Is "README.md" the same as "readme", here? I think this 1. up-case it, + 2. add `.md` is confusing, so at least (even if not dropping that support) our own usage example is consistent (?)